### DISCUSSION TITLE
360 image viewer for MAUI

### DISCUSSION THREAD

**Discussion by Brosten** [dotnet/maui#20634] [General]
Anyone knows of a 360 image (jpeg panorama) viewer for MAUI. Something like the viewer used here.
https://www.eso.org/public/images/elt-huedepohl-aug2023-drone-pano2-360/

Perhaps a webview could be used and use some web-component?

**Comment by zriman** [comment]
Check pannellum.org

**Comment by pMaske** [comment]
@Brosten Could you share if you were able to find a control that you used for 360 images in MAUI?

**Reply by Brosten** [reply]
We didn't dig that deep and no. We havn't found any such control or component.
